### PR TITLE
Handle errors

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: utils/api.js [hunk 1]
```diff
@@ -97,6 +91,12 @@ const api = {
       }
     }
 
+    if (response.statusCode >= 400) {
+      throw new Error(
+        `${response.statusCode} ${response.body.substr(0, 100)}...`
```
Comment (victorg1991) [inline utils/api.js]: 100 characters are enough to see the validation errors?

File: utils/log.js [hunk 2]
```diff
@@ -5,85 +5,49 @@ const chalk = require('chalk');
 /**
  * Logs a simple message to the console
  * @param {string} message Message content
+ * @param {object} [options]
+ * @param {boolean} [options.newLine=false]
+ * @param {boolean} [options.indent=false]
+ * @param {'info'|'success'|'error'} [options.level='info']
```
Comment (victorg1991) [inline utils/log.js]: Wouldn't be better to use constants for the level param 🤔

**Review by victorg1991 — APPROVED** [review]
I've left some comments but looks good to me :)

I really like that logging utility refactor

File: utils/api.js (hunk 1 above)
Comment (p2kmgcl) [reply utils/api.js]: LMAO I didn't see that :laughing:

File: utils/log.js (hunk 2 above)
Comment (p2kmgcl) [reply utils/log.js]: True, but because we are using typecheck it will fail if you misspell some word :)

Comment (victorg1991) [reply utils/log.js]: But does it helps to autocomplete the param?

Comment (p2kmgcl) [reply utils/log.js]: Thats true. I'm changing that

**Comment by victorg1991** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorg1991** [comment]
Wonderful :D 👏

**Pull request merged**